### ISSUE TITLE
Dependencies not shown for a meta-package

### ISSUE THREAD

**Issue by skolima** [praeclarum/FuGetGallery#19]
Example package: https://fuget.org/packages/xunit

xunit main package has no content, only dependencies - but those dependencies are not displayed by fuget.

Perhaps this is because the .nuspect is using the older format, without per-framework breakdown?

**Comment by praeclarum** [comment]
Yes, fuget only cares about real target frameworks included, not frameworks declared in your dependency list. So that fact that this package doesn't have any code means fuget never adds tfms to it. Ugh. I can special-case meta packages like this. But you'd be surprised how similar a "meta package" is to a "malformed package".
